integration-tests/src/test/java/gov/uk/address/api/client/AddressApiClient.java

```diff
@@ -153,4 +153,34 @@ private HttpResponse<String> sendHttpRequest(HttpRequest request)
             throws IOException, InterruptedException {
         return this.httpClient.send(request, HttpResponse.BodyHandlers.ofString());
     }
+
+    public HttpResponse<String> sendGetAddressesLookupRequestWithOutSessionId()
+            throws IOException, InterruptedException {
+
+        String privateApiEndpoint = this.clientConfigurationService.getPrivateApiEndpoint();
+        return sendHttpRequest(requestBuilder(privateApiEndpoint, "addresses/v2").GET().build());
+    }
+
+    public HttpResponse<String> sendNoPostCodeNoSessionIdLookUpRequest(String postcode)
```

Comment:
This method name doesn't clearly show it's intent, "send no post code" would imply that no postcode is being sent in the request, however we do in this case. 

Perhaps a better name can be "sendPostCodeLookupRequestWithNoSessionId"